tests/unit/lib/completion/completion.test.cjs

```diff
@@ -33,11 +33,12 @@ test.afterEach(() => {
   sinon.restore()
 })
 
-test('should generate a completion file', (t) => {
+test('should generate a completion file', async (t) => {
   const stub = sinon.stub(fs, 'writeFileSync').callsFake(() => {})
-  const program = createTestCommand()
   // eslint-disable-next-line n/global-require
   const { createAutocompletion } = require('../../../../src/lib/completion/generate-autocompletion.cjs')
+  const program = await createTestCommand()
```

Comment:
I had to move this down, because otherwise the mocking would fail 🙄
My guess is that if the mocked module is imported ina esm module, somehow it resets the mock again, but not sure.